### ISSUE TITLE
Workbook platform icons(Console/IOS/Android/WPF) are hardly visible in selected state in High contrast black mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#333]
### Steps to Reproduce
1.  Set the system high contrast mode to black before launching the application.
2. Launch Xamarin Workbooks application. Which invokes "Create New Workbook" window.
3. Navigate among workbook platforms (Console/IOS/Android/WPF) and observe the icon in selected state.

### Expected Behavior
Icons should be clearly visible
### Actual Behavior
Icons are hardly visible in High contrast black mode.

Note: In High contrast white mode, icon is clearly visible (Please refer screenshot)

### User Impact
Low vision user will feel difficult to see the icon.

### Screenshot
![1 hc black mode](https://user-images.githubusercontent.com/22956130/40720542-bfb96f76-6434-11e8-89b1-802945dc68ed.png)
![1 hc white mode](https://user-images.githubusercontent.com/22956130/40720543-bfe51ff4-6434-11e8-9c90-da2572d50b80.png)
![1 normal mode](https://user-images.githubusercontent.com/22956130/40720544-c0117374-6434-11e8-96f5-622c8f966aea.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbook-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
